cli/test/test-ai-fit.mjs

```
@@ -0,0 +1,288 @@
+
+test('AI_RESULT_CHROME_ROWS matches the inline result-frame chrome (~10)', () => {
+  // Must cover the inline chrome (Header + padding + title + caution + Select)
+  // so we don't route inline when it would overflow, but NOT be so large that
+  // analyses scroll on terminals where they'd fit (the old value of 20 did
+  // exactly that — see git history). 6–14 is the sane band.
+  if (AI_RESULT_CHROME_ROWS < 6 || AI_RESULT_CHROME_ROWS > 14)
+    throw new Error(`chrome reserve out of range: ${AI_RESULT_CHROME_ROWS}`)
+})
+
+test('stripAnsi removes SGR escape codes', () => {
+  const styled = `\x1b[1;36mhello\x1b[0m world`
```

ESLint

[error] Use uppercase characters for the value of the escape sequence.

(unicorn/escape-case)